doc/source/visualizing/plots.rst

```diff
@@ -930,54 +933,137 @@ Use any of the colormaps listed in the :ref:`colormaps` section.
    slc = yt.SlicePlot(ds, "z", ("gas", "density"), width=(10, "kpc"))
-   slc.set_log(("gas", "density"), False)
+   slc.set_log(("gas", "density"), False) # switch to linear scaling
    slc.save()
 
-Specifically, a field containing both positive and negative values can be plotted
-with symlog scale, by setting the boolean to be ``True`` and either providing an extra
-parameter ``linthresh`` or setting ``symlog_auto = True``. In the region around zero
-(when the log scale approaches to infinity), the linear scale will be applied to the
-region ``(-linthresh, linthresh)`` and stretched relative to the logarithmic range.
-In some cases, if yt detects zeros present in the dataset and the user has selected
-``log`` scaling, yt automatically switches to ``symlog`` scaling and automatically
-chooses a ``linthresh`` value to avoid errors.  This is the same behavior you can
-achieve by setting the keyword ``symlog_auto`` to ``True``. In these cases, yt will
-choose the smallest non-zero value in a dataset to be the ``linthresh`` value.
-As an example,
+One can switch to `symlog
+<https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.SymLogNorm.html?highlight=symlog#matplotlib.colors.SymLogNorm>`_
+by providing a "linear threshold" (``linthresh``) value.
+With ``linthresh="auto"`` yt will switch to symlog norm and guess an appropriate value
+automatically. Specifically the minimum absolute value in the image is used
+unless it's zero, in which case yt uses 1/1000 of the maximum value.
+
+.. python-script::
+
+   import yt
+
+   ds = yt.load_sample("IsolatedGalaxy")
+   slc = yt.SlicePlot(ds, "z", ("gas", "density"), width=(10, "kpc"))
+   slc.set_log(("gas", "density"), linthresh="auto")
+   slc.save()
+
+
+In some cases, you might find that the automatically selected linear threshold is not
+really suited to your dataset, for instance
+
+.. python-script::
+
+   import yt
+
+   ds = yt.load_sample("FIRE_M12i_ref11")
+   p = yt.ProjectionPlot(ds, "x", ("gas", "density"), width=(30, "Mpc"))
+   p.set_log(("gas", "density"), linthresh="auto")
+   p.save()
+
+An explicit value can be passed instead
 
 .. python-script::
 
    import yt
 
    ds = yt.load_sample("FIRE_M12i_ref11")
-   p = yt.ProjectionPlot(ds, "x", ("gas", "density"))
-   p.set_log(("gas", "density"), True, symlog_auto=True)
+   p = yt.ProjectionPlot(ds, "x", ("gas", "density"), width=(30, "Mpc"))
+   p.set_log(("gas", "density"), linthresh=(1e-22, "g/cm**2"))
    p.save()
 
-Symlog is very versatile, and will work with positive or negative dataset ranges.
-Here is an example using symlog scaling to plot a positive field with a linear range of
-``(0, linthresh)``.
+Similar to the ``zmin`` and ``zmax`` arguments of the ``set_zlim`` method, units
+can be left out in ``linthresh``.
+
+
+**Arbitrary norms**
+
+Alternatively, arbitrary matplotlib norms can be passed via the
+:meth:`~yt.visualization.plot_container.PlotContainer.set_norm` method. In that
```

Comment:
just a note that this is the corresponding link to the current docs:
https://matplotlib.org/stable/tutorials/colors/colormapnorms.html

otherwise, agreed